spec/unit/file_system_spec.rb

```diff
@@ -32,11 +38,16 @@ def with_file_content(content)
       Puppet::FileSystem.open(file, nil, 'a') { |fh| fh.write('') }
 
       expected_perms = Puppet::Util::Platform.windows? ?
-        # default Windows mode based on temp file storage
-        '2000700' :
+        # default Windows mode based on temp file storage for SYSTEM user or regular user
+        # for Jenkins or other services running as SYSTEM writing to c:\windows\temp
+        # the permissions will typically be SYSTEM(F) / Administrators(F) which is 770
+        # but sometimes there are extra users like IIS_IUSRS granted rights which adds the "extra ace" 2
+        # for local Administrators writing to their own temp folders under c:\users\USER
+        # they will have (F) for themselves, and Users will not have a permission, hence 700
```

Comment:
Thanks for the write-up @Iristyle. Agreed it's a bit confusing which seems like it could open us up to accidentally setting incorrect attributes somewhere.